domains/kaybee.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ecstaticsoul",
+        "email": "soulfulkrishna@gmail.com"
+    },
+    "record": {
+        "URL": "https://kaybee.is-a.dev"
```

Comment:
You can't put the same domain for `URL`, put the link of the website you want the to redirect to.